lib/Users.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace WorkOS;
+
+/**
+ * Class Users
+ */
+class Users
```

Comment:
It felt weird calling this `Users` in Ruby for some reason so I called it `UserManagement`. ymmv